### DISCUSSION TITLE
Input Dokumentasi Pemasukan Kas

### DISCUSSION THREAD

**Discussion by og-mfp** [noplanalderson/simasjid#3] [Q&A] [answered]
Hallo.. Saya mau nanya bang, apakah diperlukan tiap kali melakukan input data pemasukan dan pengeluaran kas masjid itu upload dokumentasi? Kalau boleh tau cara disable inputan tersebut boleh share dong bang.. Terima kasih bang..

![image](https://user-images.githubusercontent.com/24991355/194755206-5f37cb64-063d-4b9c-9c3a-e73d20756558.png)

**Comment by noplanalderson** [comment, marked as answer]
Iya itu di backendnya wajib upload. Nanti saya update agar upload dokumentasinya menjadi opsional.
Terima kasih feedbacknya

**Reply by noplanalderson** [reply]
Sudah saya update

**Reply by og-mfp** [reply]
Baik terima kasih bang
